site/docs/expressions/scalar_functions.md

```diff
@@ -1,52 +1,118 @@
 # Scalar Functions
 
-Scalar functions are a function that takes in values from a single record and produces an output value. To clearly specify the definition of functions, Substrait declares a extensible specification plus binding approach to function resolution. This ensures a clear definition of function behavior separate from a particular use of each function while avoiding operations 
+Scalar functions are a function that takes in values from a single record and produces an output value. To clearly specify the definition of functions, Substrait declares a extensible specification plus binding approach to function resolution. A scalar function signature includes the following properties:
 
-Substrait supports a number of functions. A scalar function signature includes the following properties:
+| Property               | Description                                                  | Required                            |
+| ---------------------- | ------------------------------------------------------------ | ----------------------------------- |
+| Name                   | One or more user friendly utf8 strings that are used to reference this function in languages | At least one value is required.     |
```

Comment:
What does it mean if there are multiple names?  Are they aliases of each other?

Reply:
Yeah, they are aliases.